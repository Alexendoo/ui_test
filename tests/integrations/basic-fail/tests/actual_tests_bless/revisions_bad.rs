#[cfg(foo)]
use basic_fail::add;
//@ revisions: foo bar
//@[bar] error-pattern: `main` function not found in crate `revisions_bad`

#[cfg(foo)]
fn main() {
    add("42", 3);
    //~[foo]^ ERROR: mismatched types
}
