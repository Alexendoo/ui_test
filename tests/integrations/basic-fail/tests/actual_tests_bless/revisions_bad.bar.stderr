error[E0601]: `main` function not found in crate `revisions_bad`
 --> $DIR/revisions_bad.rs:9:2
  |
9 | }
  |  ^ consider adding a `main` function to `$DIR/revisions_bad.rs`

error: aborting due to previous error

For more information about this error, try `rustc --explain E0601`.
